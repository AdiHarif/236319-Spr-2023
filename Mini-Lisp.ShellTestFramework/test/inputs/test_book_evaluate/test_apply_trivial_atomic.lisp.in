(apply-trivial-atomic 'atom 'a NIL)
(apply-trivial-atomic 'car '(a b) NIL)
(apply-trivial-atomic 'cdr '(a b) NIL)
(apply-trivial-atomic 'cons 'a 'b)
(apply-trivial-atomic 'eq 'a 'a)
(apply-trivial-atomic 'eq 'a 'b)
(apply-trivial-atomic 'set 'a 'b)
(apply-trivial-atomic 'zzz 'a NIL)
