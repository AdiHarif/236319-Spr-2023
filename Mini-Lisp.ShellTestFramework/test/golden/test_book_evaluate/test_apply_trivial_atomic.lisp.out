T
A
(B)
(A.B)
T
NIL
B
Error [SOMETHING-WENT-WRONG] in (ERROR (QUOTE SOMETHING-WENT-WRONG) ATOMIC)
